azure-pipelines.yml

```diff
@@ -33,12 +33,17 @@ steps:
   displayName: 'Use Python version'
   inputs:
     versionSpec: '$(PYTHON_VERSION)'
+    addToPath: true
     architecture: 'x64'
 
 - script: |
     python -m pip install -e ./[ci]
   displayName: 'CR-QC: Install coderedcms from local repo'
 
+- script: |
+    python -m pip freeze
+  displayName: 'CR-QC: Show installed pip packages'
```

Comment:
Does not need a CR-QC here, it is not a quality control check, just output for debugging purposes.